src/shogun/features/RandomFourierDotFeatures.cpp

```diff
@@ -96,7 +96,6 @@ SGVector<float64_t> RandomFourierDotFeatures::generate_random_parameter_vector()
 				vec[i] = std::sqrt((float64_t)1 / kernel_params[0]) *
 				         std::sqrt(2.0) * normal_dist(m_prng);
 			}
-
```

Comment:
pls no whitespace changes

Reply:
make sure to check the diff. there are now two witespace changes as you added tabs